src/components/BondLogo.jsx

```diff
@@ -1,21 +1,20 @@
-import { isBondLP } from "../helpers";
-// import "../views/Stake/stake.scss";
+import { isBondLP, getOhmTokenImage, getFraxTokenImage, getDaiTokenImage } from "../helpers";
 
 function BondHeader({ bond }) {
   const ohmAssetImg = () => {
-    return "https://raw.githubusercontent.com/sushiswap/assets/master/blockchains/ethereum/assets/0x383518188C0C6d7730D91b2c03a03C837814a899/logo.png";
+    return getOhmTokenImage();
```

Comment:
Probably generalizes to `getTokenImage(argument)` where argument is a string

Reply:
yea i think i suggested this on another pr lol. will improve the helper in next update.